book/animations.md

```diff
@@ -894,8 +881,9 @@ layout and raster steps if the display list didn't change much between frames.
 
 The algorithm presented here is a simplified version of what
 Chromium actually implements. For more details and information on how Chromium
-implements these concepts see [here][renderingng-dl] and
-[here][rendersurface]; other browsers do something broadly similar. Chromium's
+implements these concepts see [blog][renderingng-dl]
+[posts][rendersurface]
+on the Chrome developer blog; other browsers do something broadly similar. Chromium's
```

Comment:
I don't care that much how exactly we phrase this, but "here" and "this" don't work in print.